recipes/secretaria

```diff
@@ -0,0 +1 @@
+(secretaria :fetcher git :url "https://bitbucket.org/shackra/secretaria.el")
```

Comment:
I'm using Git on Bitbucket.org, so as the MELPA documentatio says, I should be using git as fetcher.